package.json

```diff
@@ -15,7 +15,8 @@
     "marked": "^4.0.17",
     "npm-stats": "1.2.0",
     "semver": "5.0.1",
-    "superagent": "5.3.1"
+    "superagent": "5.3.1",
+    "toml": "3.0.0"
```

Comment:
question: isn't there a better dependency? This one hasn't been updated for 6 years

Reply:
Mmh, would this one work better for you? https://www.npmjs.com/package/smol-toml

(The search is here: https://www.npmjs.com/search?q=toml, we can rank it by Optimal -IdK what this is-, Popularity, Quality or Maintenance)

Reply:
Well, it seems like the most recent-decent packages are not compatible with our Node.js requirement for semver, which is `16.x` (e.g. `smol-toml` requires Node.js ≥ 18).
I managed to work with `@iarna/toml` which is 4yrs old. I'm not sure that's really better.
WDYT?

Reply:
Arf we really need to either work on https://github.com/Scalingo/semver.io/issues/16 or keep this repo up-to-date. I'll ping Méderic on #16. Can you also open an issue so that we update the Node.js version?

Reply:
I filed https://github.com/Scalingo/semver.io/issues/87